spec/requests/api/braintree_spec.rb

```diff
@@ -222,6 +222,20 @@
               end
             end
 
+            xit 'works' do
```

Comment:
naturally, this shouldn't be there. I'm on another branch, mid thought, would you be so kind as to remove the spec, and make any other changes you see fit. :tulip: